balancer/balancer.go

```diff
@@ -329,7 +329,8 @@ type PickResult struct {
 	SubConn SubConn
 
 	// Done is called when the RPC is completed.  If the SubConn is not ready,
-	// this will be called with a nil parameter.  If the SubConn is not a valid
+	// this will be called with an empty DoneInfo: meaning that Pick did not
+	// error but no bytes were sent nor received.  If the SubConn is not a valid
```

Comment:
Grammar nit: it should be "but no bytes were sent _or_ received" here, not "nor".

Adding an "either" or "neither" to the sentence, only "either" makes sense:

"but no bytes were _either_ sent _or_ received" vs
"but no bytes were _neither_ sent _nor_ received".

If you say "but bytes were neither sent nor received", that would be correct, too, but a bit awkward.